frappe/model/db_query.py

```diff
@@ -97,6 +97,7 @@ def execute(
 		strict=True,
 		pluck=None,
 		ignore_ddl=False,
+		*,
 		parent_doctype=None,
```

Comment:
`parent_doctype` will be removed from here and added to `DatabaseQuery.__init__` - in another PR 😅